core/src/main/java/com/opensymphony/xwork2/ognl/SecurityMemberAccess.java

```diff
@@ -105,19 +105,25 @@ public void restore(Map context, Object target, Member member, String propertyNa
     public boolean isAccessible(Map context, Object target, Member member, String propertyName) {
         LOG.debug("Checking access for [target: {}, member: {}, property: {}]", target, member, propertyName);
 
-        if (checkEnumAccess(target, member)) {
-            LOG.trace("Allowing access to enum: target [{}], member [{}]", target, member);
-            return true;
+        final int memberModifiers = member.getModifiers();
+
+        if (!checkPublicMemberAccess(memberModifiers)) {
+            LOG.trace("Access to non-public [{}] is blocked!", member);
```

Comment:
Should we warn users about that?

Reply:
Hello @lukaszlenart .
I'm not sure what @aleksandr-m  thinks, but the log-level for failure due to non-public members might be a "balancing act".  
My understanding is that prior to this the framework would just return false for non-public elements (so downstream processing would determine whether any logging happened in such cases) ?
A _warn_ for this condition might end up being noisy for some apps when migrating apps from 2.5.x to 2.6.  Maybe a level of _debug_ (or even _info_) might be a better balance in this case ?

Reply:
I meant, accessing non-public methods/fields is a visible sign that something is wrong. And even before the change, access to non-publics was blocked so I see no problems with migrations.

Reply:
OK ... I think I see your point.  So maybe a warn-level for 2.6 for failing the initial public member check (but if this check was back-ported to 2.5.x it would be trace or debug ...).

Reply:
Ok, but I still don't see a valid case, even in Struts 2.5.x you are not allowed to access non-public methods, you can only switch on/off access to static methods.

Reply:
I prefer to warn i.e. a `LOG.warn` statement - it warns user about a misuse in underlying app or about an attack try on it.

Reply:
Let's make it simple. :)

@lukaszlenart 

1. Remove log at this line
2. Change log level to `warn`
3. Leave as it is now

Choose one.